jsk_panda_robot/panda_eus/euslisp/end-coords_publisher.l

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env roseus
+
+(require "package://panda_eus/euslisp/dual_panda-interface.l")
+
+(ros::roseus-add-msgs "std_msgs")
+(ros::roseus-add-msgs "sensor_msgs")
+(ros::roseus-add-msgs "geometry_msgs")
+
+(ros::roseus "pose-stamped-publisher")
+
+(setq *robot* (dual_panda))
+
+(ros::advertise "/dual_panda/end_coords/rarm" geometry_msgs::PoseStamped 1)
+(ros::advertise "/dual_panda/end_coords/larm" geometry_msgs::PoseStamped 1)
+
+(defvar *joints* nil)
+
+(defun joints-cb (msg)
+  (setq buf (send msg :position))
+  (setq buf (coerce buf cons))
+  (setq l (- (length buf) 1))
+  (setq i l)
+  (setq ans nil)
+  (print "publishing end-coords...")
+  (while (<= 0 i)
+    (if (and (not (= i 7)) (not (= i 8)) (not (= i 16)) (not (= i 17)))
+	(if (not ans)
+	    (setq ans (cons (elt buf i) nil))
+	  (setq ans (cons (elt buf i) ans))))
+    (setq i (- i 1)))
+  (setq *joints* (coerce ans float-vector)))
+
+(ros::rate 10)
```

Comment:
Is there any reason that the rate is 10?
Maybe another way is to use set some rosparam like `/dual_panda/end_coords_pub_hz` and set this value in  startup launch file.
https://euslisp-docs.readthedocs.io/en/latest/roseus/roseus/#rosget-param
http://wiki.ros.org/roslaunch/XML/rosparam

Reply:
I set rosparam '/dual_panda/end_coords_pub_hz' and set this value in dual_panda.launch. Besides, changed '(ros::rate 10)' to '(ros::rate (ros::get-param "/dual_panda/end_coords_pub_hz")' , then we can change rosrate when launch. Default value is 30. 
https://github.com/softyanija/jsk_robot/commit/75d8b0cb6dc3cd0c1cafb8b1686465a376110ddd
https://github.com/softyanija/jsk_robot/commit/fd28f69473e262e5bbe056335b240977c8c1a962